xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-theme/xwiki-platform-flamingo-theme-ui/src/main/resources/FlamingoThemesCode/WebHomeSheet.xml

```diff
@@ -173,7 +173,7 @@
   #set ($isCurrentTheme = ($currentThemeDoc.documentReference == $themeReference))
   #set ($themeDoc = $xwiki.getDocument($themeReference))
   #set ($themeObj = $themeDoc.getObject('FlamingoThemesCode.ThemeClass'))
-  (% class="theme #if($isCurrentTheme) current-theme#end" %)
+  * (% class="theme #if($isCurrentTheme) current-theme#end" %)
```

Comment:
Why is it better to turn those div elements into list elements?

Reply:
Because what it shows is a list of themes. Tbh it's not the main point of this ticket, but IMO it just made sense and made it easier to style these elements because we have a more precise architecture.

In addition, using semantic elements can make life easier for assistive tech users. Eg. a blind user navigates to the start of the list, it reads out `unordered list, 25 elements` and now they have shortcuts to navigate quickly to the end of the list if that's not what's interesting them.

Reply:
It could be argued that it's more like a select with a bunch of options. I was lazy here and took the easy implementation with somewhat correct semantics :D

Reply:
ok, I wasn't aware `li` are the most semantically accurate element for this kind of thing.
Thanks for the explanation.